modules/apps/friendly-url/friendly-url-service/src/main/java/com/liferay/friendly/url/model/impl/FriendlyURLEntryImpl.java

```diff
@@ -5,22 +5,66 @@
 
 package com.liferay.friendly.url.model.impl;
 
+import com.liferay.asset.kernel.model.AssetCategory;
+import com.liferay.asset.kernel.model.AssetEntry;
+import com.liferay.asset.kernel.service.AssetEntryLocalServiceUtil;
 import com.liferay.friendly.url.model.FriendlyURLEntry;
+import com.liferay.friendly.url.model.FriendlyURLEntryLocalization;
 import com.liferay.friendly.url.service.FriendlyURLEntryLocalServiceUtil;
+import com.liferay.petra.string.StringBundler;
 import com.liferay.petra.string.StringPool;
 import com.liferay.portal.kernel.exception.PortalException;
 import com.liferay.portal.kernel.util.LocalizationUtil;
 import com.liferay.portal.kernel.util.Validator;
 
 import java.util.Collection;
 import java.util.Iterator;
+import java.util.List;
 import java.util.Map;
 
 /**
  * @author Pavel Savinov
+ * @author Roberto Díaz
  */
 public class FriendlyURLEntryImpl extends FriendlyURLEntryBaseImpl {
 
+	@Override
+	public String getCategorizedUrlTitle(String languageId) {
+		FriendlyURLEntryLocalization friendlyURLEntryLocalization =
+			FriendlyURLEntryLocalServiceUtil.fetchFriendlyURLEntryLocalization(
+				getPrimaryKey(), languageId);
+
+		if (friendlyURLEntryLocalization == null) {
+			return StringPool.BLANK;
+		}
+
+		String urlTitle = friendlyURLEntryLocalization.getUrlTitle();
```

Comment:
This is duplicating the implementation of the super class. Shouldn't we call `super.getUrlTitle(languageId)` instead?

Reply:
Done :)